tests/ClientTest.php

```diff
@@ -210,6 +207,24 @@ public function join($timeout = null)
     }
 }
 
+class Dummy_Serializer implements \Raven\SerializerInterface
```

Comment:
This seems bad because there is logic code in a test that doesn't reflect what the real implementation of the serializer does. You should use a mocked object instead